rs/execution_environment/src/execution/install_code/tests.rs

```diff
@@ -91,7 +91,7 @@ fn install_code_fails_on_invalid_memory_allocation() {
     assert_eq!(
         format!(
             "MemoryAllocation expected to be in the range [0..{}], got 18_446_744_073_709_551_615",
-            candid::Nat((MAX_STABLE_MEMORY_IN_BYTES + MAX_WASM_MEMORY_IN_BYTES).into())
+            candid::Nat((MAX_STABLE_MEMORY_IN_BYTES + MAX_WASM64_MEMORY_IN_BYTES).into())
```

Comment:
Should this instead use `MAX_MEMORY_ALLOCATION` from `ic_types` instead to be more robust in the future?